scripts/build-docs.ts

```diff
@@ -1,27 +1,22 @@
 import * as path from "path";
 import * as fs from "fs-extra";
 import { Application, TSConfigReader, TypeDocOptions } from "typedoc";
+import * as yargs from 'yargs';
 
 const OUTPUT_DIR = "dist/docs";
 const INDEX_FILE = "doc/index.md";
-
-const GITHUB_FORMAT = 'github';
-const PC_DEV_CENTER_FORMAT = 'purecloudDevCenter';
-const SUPPORTED_DOC_OUTPUT_FORMATS = [GITHUB_FORMAT, PC_DEV_CENTER_FORMAT];
-const docMdOutputFormat = process.env.DOC_MD_OUTPUT_FORMAT || PC_DEV_CENTER_FORMAT;
-if (SUPPORTED_DOC_OUTPUT_FORMATS.indexOf(docMdOutputFormat) < 0) {
-    console.error(`Unknown MD Output Format Specified: '${docMdOutputFormat}'`);
-    process.exit(1);
-}
-
-let flagDocsAsPreview = true;
-if (process.env.FLAG_DOCS_AS_PREVIEW) {
-    ['false', 'f', '0'].forEach(currFalseValue => {
-        if (process.env.FLAG_DOCS_AS_PREVIEW === currFalseValue) {
-            flagDocsAsPreview = false;
-        }
-    });
-}
+const SUPPORTED_DOC_OUTPUT_FORMATS = ['github', 'purecloudDevCenter'] as const;
+
+const { preview: flagDocsAsPreview, format: docMdOutputFormat } = yargs
+    .option('preview', {
+        type: 'boolean',
+        default: false
```

Comment:
Conscious decision to switch the default?  Seems reasonable give we're live but wanted to make sure.

Reply:
Yeah, pretty sure it was intentional / think it makes reasonable sense that we would have to explicitly mark as a "preview" version.